src/Sql/dbo/Tables/Provider.sql

```diff
@@ -0,0 +1,19 @@
+﻿CREATE TABLE [dbo].[Provider] (
+    [Id]                UNIQUEIDENTIFIER NOT NULL,
+    [Name]              NVARCHAR (50)    NOT NULL,
+    [BusinessName]      NVARCHAR (50)    NULL,
+    [BusinessAddress1]  NVARCHAR (50)    NULL,
+    [BusinessAddress2]  NVARCHAR (50)    NULL,
+    [BusinessAddress3]  NVARCHAR (50)    NULL,
+    [BusinessCountry]   VARCHAR (2)      NULL,
+    [BusinessTaxNumber] NVARCHAR (30)    NULL,
+    [BillingEmail]      NVARCHAR (256)   NOT NULL,
+    [Plan]              NVARCHAR (50)    NOT NULL,
```

Comment:
doesn't seem like a provider will have a Plan or PlanType, or even Seats since these are directed to each Organization they may create/manage...

Reply:
The Parent Asana ticket mentions `Contain a total seat-count` but perhaps we consider that to be outside the scope of this MVP. Just noticed it's under **Previous**, so it might be out of date.

Reply:
Replied in Asana.